[2025-04-28T10:35:22.818+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2025-04-27T00:00:00+00:00 [queued]>
[2025-04-28T10:35:23.179+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2025-04-27T00:00:00+00:00 [queued]>
[2025-04-28T10:35:23.205+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2025-04-28T10:35:23.214+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2025-04-28T10:35:23.225+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2025-04-28T10:35:23.534+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2025-04-27 00:00:00+00:00
[2025-04-28T10:35:23.894+0000] {standard_task_runner.py:55} INFO - Started process 120 to run task
[2025-04-28T10:35:23.896+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2025-04-27T00:00:00+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp6epwy74m']
[2025-04-28T10:35:23.926+0000] {standard_task_runner.py:83} INFO - Job 239: Subtask create_table
[2025-04-28T10:35:25.386+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2025-04-27T00:00:00+00:00 [running]> on host fae2b1eff5d6
[2025-04-28T10:35:27.118+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2025-04-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-04-27T00:00:00+00:00
[2025-04-28T10:35:27.619+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2025-04-28T10:35:27.811+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstName TEXT NOT NULL,
                lastName TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2025-04-28T10:35:27.829+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2025-04-28T10:35:28.072+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20250427T000000, start_date=20250428T103522, end_date=20250428T103528
[2025-04-28T10:35:29.841+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2025-04-28T10:35:30.384+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
